src/app/components/events/events.component.html

```diff
@@ -1,9 +1,6 @@
+<p class="page-title">Upcoming Event</p>
```

Comment:
@Bawanthathilan in here I used the same CSS class which I used for about page but font sizes are different?
![Screenshot 2021-04-28 at 19 04 17](https://user-images.githubusercontent.com/19349315/116412733-98d8db00-a854-11eb-83a3-4f09c8dc25ae.png)
![Screenshot 2021-04-28 at 19 04 25](https://user-images.githubusercontent.com/19349315/116412779-a2fad980-a854-11eb-8c89-c4d91913e356.png)

Reply:
font family are different 🙄

Reply:
@Bawanthathilan can you fix that?